.github/workflows/unittests.yml

```diff
@@ -25,6 +25,30 @@ jobs:
           python -m pip install -U pip setuptools wheel
           pip install pytest requests-mock
           pip install -e ./api/python/cell_census/
-      - name: Test with pytest
+      - name: Test with pytest (API)
         run: |
           pytest ./api/python/cell_census/tests/
+
+  unit_tests_builder:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ["3.9", "3.10"]
```

Comment:
Tiny, tiny nit:  i'm not sure we need to test on more than one version, as we are the only users.  So strictly from a "how fast is our CI", I suggest a simple test matrix.

Reply:
I only set 3.10